helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -94,6 +94,8 @@ monitoring:
         - 443
 opensearch:
   enabled: true
+  client:
+    enabled: false
```

Comment:
Oops, must've slipped through 

Could you also add
```suggestion
  client:
    enabled: false
  data:
    enabled: false
```

Reply:
but i dont use, data.enabled anywhere 

do you want me to use it ?